Napzakmarket-iOS/Napzakmarket-iOS/Presentation/Mypage/View/MarketView.swift

```diff
@@ -0,0 +1,219 @@
+//
+//  MarketView.swift
+//  Napzakmarket-iOS
+//
+//  Created by 어진 on 1/13/25.
+//
+
+import SwiftUI
+
+struct MarketView: View {
+    @Environment(\.dismiss) private var dismiss
+    @EnvironmentObject private var tabBarState: TabBarStateModel
+
+    @State private var tags: [Tag] = MarketMockData.tags
+    
+    @State private var selectedIndex = 0
+    @State private var sellProducts = ProductModel.sellDummyList()
+    @State private var buyProducts = ProductModel.buyDummyList()
+   
+    private let columns = [
+        GridItem(.flexible(), spacing: 15),
+        GridItem(.flexible(), spacing: 15)
+    ]
+   
+    var body: some View {
+        VStack(spacing: 0) {
+            navigationSection
+            profileSection
+           
+            NZSegmentedControl(
+                selectedIndex: $selectedIndex,
+                tabs: ["팔아요", "구해요", "후기"],
+                spacing: 15
+            )
+            .padding(.vertical, -65)
+            .padding(.top, 70)
+            .padding(.bottom, -15)
```

Comment:
P1

```suggestion
            .frame(height: 46)
            .padding(.top, 20)
```

패딩값 피그마랑 조금 다른 것 같아요! 이렇게 바꿔주면 조을듯